packages/orbit-components/src/SmartPassIllustrations/index.js.flow

```diff
@@ -0,0 +1,26 @@
+// @flow
+import * as React from "react";
+
+import type { Globals } from "../common/common.js.flow";
+
+export type Sizes = "extraSmall" | "small" | "medium" | "large" | "display";
+
+export type Props = {|
+  +size?: Sizes,
+  +primary?: string,
+  +secondary?: string,
+  +title?: string,
+  +description?: string,
+  +ariaLabelledby?: string,
+  ...Globals,
+|};
+
+type SmartPassIllustration = React.ComponentType<Props>;
+
+declare export var SmartPassV1: SmartPassIllustration;
+declare export var SmartPassV2: SmartPassIllustration;
+declare export var SmartPassV3: SmartPassIllustration;
+declare export var SmartPassV4: SmartPassIllustration;
+declare export var SmartPassV5: SmartPassIllustration;
+
+declare export default SmartPassIllustration;
```

Comment:
Considering that `packages/orbit-components/src/SmartPassIllustrations/index.js` doesn't have a default export, is this type declaration correct? 🤔